package.json

```diff
@@ -75,7 +75,7 @@
     "prop-types": "15.7.2",
     "react-is": "16.9.0",
     "react-required-if": "1.0.3",
-    "react-select": "3.0.4",
+    "react-select": "3.0.5",
```

Comment:
I was hoping this would remove the need for the componentWillReceiveProps warning squashing, but there's still a problem with one of react-select's deps

https://github.com/JedWatson/react-input-autosize/issues/163